content/docs/2.1/scalers/openstack-swift.md

```diff
@@ -0,0 +1,155 @@
++++
+title = "OpenStack Swift"
+layout = "scaler"
+availability = "v2.1+"
+maintainer = "Community"
+description = "Scale applications based on the count of objects in a given OpenStack Swift container."
+go_file = "openstack_swift_scaler"
++++
+
+### Trigger Specification
+
+This specification describes the `swift` trigger for OpenStack Swift containers. It scales based on the count of objects in a given container.
+
+```yaml
+triggers:
+- type: swift
+  metadata:
+    swiftURL: http://localhost:8080/v1/b161dc815cd24bda84d94d9a0e73cf78  # Required
+    containerName: my-container # Required
+    objectCount: "2"  # Optional
+    objectPrefix: "my-prefix" # Optional
+    objectDelimiter: "/"  # Optional 
+    objectLimit: "10" # Optional
+    onlyFiles: "true" # Optional
+    timeout: "2"  # Optional
+```
+
+> Please, always provide the protocol (http or https) when specifying URLs. This is needed due to Go URL parsing issues :sweat_smile:
+
+**Parameter list:**
+
+- `swiftURL` - The URL to query the Swift API. It must contain the hostname, the Swift port, the API version and the account ID. The pattern is: `http://<host>:<swift_port>/<swift_version>/<account_id>`
+- `containerName` - Name of Swift container in an OpenStack account
+- `objectCount` - Average target value to trigger scaling actions. If not specified, the default value is `2`
+- `objectPrefix` - Prefix for the object. Only objects with this prefix will be returned. Use this prefix to specifiy sub paths for the objects, if required. If not specified, the default value is `""`.
+- `objectDelimiter` - Delimiter for identifying the object prefix. It is the character used to split object names. The default value is `""`.
+- `objectLimit` - The maximum amount of objects returned by the API. By default, the Swift API only returns up to 10000 names. The default value is `""`, which will use the Swift API standard limit. Negative values defaults to the Swift API standard limit.
+- `onlyFiles` - Boolean value that specifies if the scaler should be triggered only by the amount of files, without considering folders. Inside a container, one can have files and folders. Folders (empty or not) are counted as objects, just as files are. If one wants to scale based on only files, this parameter must be set to `true`. The default value is `false`.
+- `timeout` - The timeout, in seconds, for the HTTP client requests that will query the Swift API. If not specified, the default value is `30` seconds. This must be an integer value.
+
+For more information about `prefix`, `delimiter`, and `limit`, please, refer to the [Object Store API](https://docs.openstack.org/api-ref/object-store/).
+
+### Authentication Parameters
```

Comment:
Can you please align with other scalers by following this pattern please?

https://github.com/kedacore/keda-docs/blob/master/content/docs/2.1/scalers/azure-service-bus.md#L44-L50

Typically we first list all auth options and then dive into the examples.

Reply:
Sure! It is done by b9e4bb5.